Tests/GRDBTests/DatabaseQueueTests.swift

```diff
@@ -2,6 +2,7 @@ import XCTest
 import Dispatch
 import GRDB
 
+#if canImport(Darwin) // needed for __dispatch_get_global_queue
```

Comment:
I understand that this pull request is a draft, and you were not aiming at polishing it.

I'm thus sure you agree that the tests that do not use `__dispatch_get_global_queue` should keep on running.

Usually, I keep all tests, and disable them with `XCTSkip`, in a way or another:

```swift
func test_foo() throws {
#if condition
    // the test
#else
    throw XCTSkip("XXX is not available")
#endif
}
```

This will be needed before we merge.

- DatabaseMigratorTests
- DatabasePoolConcurrencyTests
- DatabaseQueueTests
- DatabaseRegionObservationTests
- DatabaseSnapshotTests

I spare you the tests that are not intended to run and are only testing if some code is accepted by the compiler ("Test passes if it compiles" in DatabaseRegionObservationTests et al.)

Reply:
OK, I've switched to throwing `XCTSkip` rather than wholesale #if'ing out the test cases when a required framework is unavailable in [289f9fd].